### PR TITLE
Overhaul

### LINKED ISSUE #1: Create a `login` command
Create a login command which gets user's credentials, authorises with Appknox and stores the token in `appknox.json` file in home directory.

### LINKED ISSUE #21: Remove Submit url method

### PR DESCRIPTION
**Pull request by viren-nadkarni**
- More *pythonic*
- Improved documentation
- Sessions (finally fixes #1!)
- MFA support

Also closes #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,119 @@
...
+                             When sharing a provider's data with other apps, access control should be carefully implemented to prohibit unauthorized access to sensitive data.")
 ```
 
-### Uploading your App
+## Command-line interface
 
 ```
-appknox upload /path/to/your/android.apk
-appknox upload /path/to/your/ios.ipa
+$ appknox login -h https://api.appknox.com
```
Comment (dhilipsiva) [inline README.md]: @viren-nadkarni Question: api.appknox.com is the default right?

Comment (viren-nadkarni) [inline README.md]: yessir!

Comment (dhilipsiva) [inline README.md]: Then can we remove that from docs? I just want people to get started with as much less lines as possible 😛

Comment (dhilipsiva) [inline README.md]: Or maybe we can add it some where else.

Comment (viren-nadkarni) [inline README.md]: sure, I'll remove it from the readme and put it in sphinx docs

File: README.md [hunk 2]
```diff
@@ -1,77 +1,119 @@
...
+  Command line wrapper for the Appknox API
+
+Options:
+  -v, --verbose  Specify log verbosity.
+  --help         Show this message and exit.
+
+Commands:
+  analyses       List analyses for file
+  dynamic_start  Start dynamic scan for file
+  dynamic_stop   Stop dynamic scan for file
```
Comment (ChillarAnand) [inline README.md]: This can be merged to a single command with a flag?

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,4 @@
-click==4.1
-pkginfo==1.2.1
-pypandoc==1.0.1
-requests==2.7.0
-twine==1.5.0
-wheel==0.24.0
+click
+requests
+slumber
+tabulate
```
Comment (ChillarAnand) [inline requirements.txt]: Can you preserve versions? This might cause problems with compatibility issues.

File: README.md (hunk 2 above)
Comment (viren-nadkarni) [inline README.md]: true, subcommands would have made more sense, but click doesn't have a straightforward way to implement nested subcommands

File: requirements.txt (hunk 3 above)
Comment (viren-nadkarni) [inline requirements.txt]: :+1:

**Pull request merged**